tests/unit/Runner/DefaultTestResultCacheTest.php

```diff
@@ -0,0 +1,34 @@
+<?php declare(strict_types=1);
+/*
+ * This file is part of PHPUnit.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace PHPUnit\Runner;
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers \PHPUnit\Runner\DefaultTestResultCache
+ * @small
+ */
+final class DefaultTestResultCacheTest extends TestCase
+{
+    /**
+     * @var DefaultTestResultCache
+     */
+    private $subject;
+
+    protected function setUp(): void
+    {
+        $this->subject = new DefaultTestResultCache();
+    }
+
+    public function testGetTimeForInexistentTestNameReturnsZero(): void
```

Comment:
The word _inexistent_ is _non-existent_, what do you think about using _non-existent_ instead?

Also, the test method name reflects the current behavior, perhaps it makes sense to add `AsFloat` as suffix?